src/pages/contact-us.jsx

```diff
@@ -179,14 +182,14 @@ function ContactUs({ t, choices }) {
 }
 
 export async function getStaticProps({ locale }) {
-    const typeOfContactData = await getDocument("type_of_contact");
+    const row = await getDocument("contact");
     // Take only locale row
-    const typeOfContactDataByLang = typeOfContactData.find(
-        (data) => data.id === locale
-    );
+    const currentLangInfo = row.find((data) => data.id === locale);
+
+    const reasons = currentLangInfo.typeContact;
 
-    // Pick up value field in db, and convert to array to pass
-    const choices = Object.entries(typeOfContactDataByLang.value);
+    // convert ||-separated string in db address to array.
+    const address = currentLangInfo.address.split("||");
```

Comment:
- Add exception handling if the document does not exist or the value returned is null.